test/sanity/pep8/skip.txt

```diff
@@ -0,0 +1 @@
+lib/ansible/module_utils/distro/_distro.py
```

Comment:
Include a comment explaining why this is being skipped so someone doesn't try fixing it in the future.